pkg/kubernetes/istio_client.go

```diff
@@ -0,0 +1,91 @@
+package kubernetes
+
+import (
+	"fmt"
+
+	"k8s.io/client-go/rest"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const (
+	istioObjGroupName    = "config.istio.io"
+	istioObjGroupVersion = "v1alpha2"
+
+	handlerKind   = "handler"
```

Comment:
Right, I must have messed up the order I committed them in. Will note in future unless you think it needs to be reorganised here?

Reply:
It's a very nice-to-have property once you find yourself debugging and looking for the point where an issue was introduced, but let's just have it in mind next time, as it might be a bit too much extra work now.